sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -495,13 +495,16 @@ case class TruncateTableCommand(
 }
 
 /**
- * Command that looks like
+ * Commands can of the following forms.
  * {{{
  *   DESCRIBE [EXTENDED|FORMATTED] table_name partitionSpec?;
+ *   DESCRIBE QUERY <query>
+ *   DESCRIBE <query>
```

Comment:
`DESCRIBE QUERY <query>` and `DESCRIBE <query>` equal to `DESCRIBE [QUERY] <query>`?

Reply:
Yeah... would change.